src/Surfnet/ServiceProviderDashboard/Infrastructure/Manage/Factory/SaveCommandFactory.php

```diff
@@ -109,6 +111,7 @@ public function buildOidcngCommandByManageEntity(ManageEntity $manageEntity, str
         $command->setAdministrativeContact(Contact::from($metaData->getContacts()->findAdministrativeContact()));
         $command->setTechnicalContact(Contact::from($metaData->getContacts()->findTechnicalContact()));
         $command->setSupportContact(Contact::from($metaData->getContacts()->findSupportContact()));
+        $command->setIsCopy($isCopy);
```

Comment:
Again, no real gain in allowing the isCopy to be set here rigth? It is never used outside of the default false value in this context IIRC.

Reply:
It is, see above ;-)